### ISSUE TITLE
Docs fail to display code references

### ISSUE THREAD

**Issue by Pikanchion** [RiptideNetworking/Riptide#114]
Code references to primitive types fail to display in the Docs: `<see cref="byte"/>`, `<see cref="ushort"/>`, etc.

For instance: [Message.AddByte](https://github.com/RiptideNetworking/Riptide/blob/main/RiptideNetworking/RiptideNetworking/Message.cs#L241) reads `/// <summary>Adds a single <see cref="byte"/> to the message.</summary>`, but the corresponding [Docs page](https://riptide.tomweiland.net/api/Riptide.Message.AddByte.html) instead displays
> Adds a single to the message.

It seems internal references display as intended, such as the `/// <summary>The send mode of a <see cref="Message"/>.</summary>` on line 14 of the same file and its [relevant page](https://riptide.tomweiland.net/api/Riptide.MessageSendMode.html), which correctly reads
> The send mode of a [Message](https://riptide.tomweiland.net/api/Riptide.Message.html).

**Comment by tom-weiland** [comment]
What's very strange is that when I build the documentation locally, it doesn't have this issue...

I don't think I made any changes to the docs (which would affect this) since the last time the site was updated. Either I'm mistaken or something works differently when building the docs locally compared to the workflow that builds the public site 🤔

Either way, I guess we won't find out until the next update for Riptide is ready, as I think it would be confusing to have the docs include features/information which aren't released/usable yet. It'd probably be good to set things up in a way that makes it possible to improve & tweak the live docs while excluding stuff related to unreleased feature changes & additions (perhaps with a separate branch for the docs of the latest release), but I could also see that getting a bit messy to maintain. Let me know if you've got any suggestions!
